steward/src/error.rs

```diff
@@ -47,10 +47,22 @@ pub enum ErrorKind {
     ProviderError,
     /// Strategy Provider call error
     #[error("SP call error")]
-    SPCallError,
+    SPCall,
```

Comment:
Should `SPCall`/`GovernanceCall` be `SPCallError` etc. given that these are error types? Same below for `Cache`.

Reply:
They can be, my thinking was it is a little redundant when referencing it as `ErrorKind::SomethingError` like we usually do but if it were to be imported and used without the full path it would be confusing I guess.